ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -44,39 +45,79 @@ describe('AdminInstitutionEditSpec', () => {
       .toContain('Display name must be 80 characters or less');
   });
 
-  it('should default DUA to Master', async() => {
+  it('should default DOMAINS to ADDRESSES in Registered Tier requirement', async() => {
     urlParamsStore.next({
       institutionId: 'Verily'
     });
     const wrapper = component();
     await waitOneTickAndUpdate(wrapper);
     expect(wrapper).toBeTruthy();
 
-    const agreementTypeDropDown = wrapper.find('[data-test-id="agreement-dropdown"]');
-    expect(agreementTypeDropDown.first().props().value).toBe(InstitutionMembershipRequirement.DOMAINS);
+    const rtAgreementTypeDropDown = wrapper.find('[data-test-id="rt-agreement-dropdown"]');
+
+    expect(rtAgreementTypeDropDown.first().props().value).toBe(InstitutionMembershipRequirement.DOMAINS);
+  });
+
+  it('should default DOMAINS to ADDRESSES in Controlled Tier requirement', async() => {
+    urlParamsStore.next({
+      institutionId: 'Verily'
+    });
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(wrapper).toBeTruthy();
+
+    const ctEnabledToggle = wrapper.find('[data-test-id="ct-enabled-switch"]').first().instance() as InputSwitch;
+    ctEnabledToggle.props.onChange({
+      originalEvent: undefined,
+      value: true,
+      target: {name: 'name', id: '', value: true}
+    });
+    await waitOneTickAndUpdate(wrapper);
+
+    const ctAgreementTypeDropDown = wrapper.find('[data-test-id="ct-agreement-dropdown"]');
+    expect(ctAgreementTypeDropDown.first().props().value).toBe(InstitutionMembershipRequirement.DOMAINS);
+  });
+
+  it('should hide/show controlled card when controlled tier disabled/enabled', async() => {
```

Comment:
is there a test for the state where the institution has CT enabled in the DB, so it should appear enabled here already?  we should add one if so

Reply:
I'm not seeing any "edit mode" tests ... please add some of these.

Reply:
Added few tests for 
1: show CT card by default
2: Not sure what is 'edit mode test', I added a test to make some modifications and click SAVE. 
PTAL!